src/services/activation/services/ActivationModelService.js

```diff
@@ -0,0 +1,30 @@
+const feathersMongooseService = require('feathers-mongoose');
+const auth = require('@feathersjs/authentication');
+const {
+	iff, isProvider, disallow,
+} = require('feathers-hooks-common');
+const { activationModel } = require('../model');
+const { enableQuery, enableQueryAfter } = require('../../../hooks');
+
+const activationModelHooks = {
+	before: {
+		all: [auth.hooks.authenticate('jwt'), iff(isProvider('external'), disallow())],
```

Comment:
check redirect works when clicking the activation link in logged out mode

Reply:
-> SC-5236